### ISSUE TITLE
Click handlers bound to parent not to owner.

### ISSUE THREAD

**Issue by michael** [substance/substance#312]
``` js
// from AddBibItemPanel
$$(Panel).ref('panelEl').append(
  $$('button').addClass('button float-right')
    .append("Search")
    .on('click', this.startSearch)
)
```

in the `startSearch()` handler `this` is bound to the panel component, not the AddBibItemPanel.

**Comment by michael** [comment]
Same goes for refs! That's actually critical. We never ran into it since we didn't use wrapper panels that often.

**Comment by obuchtala** [comment]
Will take a look.

**Comment by obuchtala** [comment]
This is solved in experimental branch 'issue-312'.
I had to make breaking changes to the Component API, so we need to re-iterate first on demos and lens.

**Comment by obuchtala** [comment]
Inadvertently closed via commit.

**Comment by michael** [comment]
fixed via #346
